OS X: Use Retina Icons

ryanbaer opened the pull request:
Fix for #25

- Add @ 2x and @ 3x icons
- Add dependency: retina.js for tomato icon in index.html
- Add missing dependency to bower.json: jquery-ui

G07cha commented:
Hi, thanks for your effort, but seems like it could be done easier just by adding "IconTemplate@2x.png", see https://github.com/maxogden/menubar#options

Also thanks for pointing out missing jquery-ui package.

ryanbaer commented:
Hello,

Thanks again for the quick response.

Let me clarify what I've done. There are two separate issues addressed in this PR.

The first is, as you've pointed out, to add @ 2x to fix the menubar icon. I added @ 3x for good measure. This is all that was required to fix the menubar on retina displays.

The second issue came when I noticed that the tomato icon used on index.html does not adapt to retina displays - it still comes out fuzzy and pixelated and is more noticeable with the menubar icon fixed. To remedy this, I added a dependency, retina.js, which correctly detects the screen resolution and automatically uses the appropriate template icon.

G07cha commented:
Oh, ok, now I get, thanks for explanation and your effort too, going to release new version soon!
